src/index.html

```diff
@@ -1,13 +1,319 @@
+              >
+            </a>
+
+            <h3 class="feature__title">Hydraulic disc brakes & Lightweight</h3>
+
+            <p class="feature__description">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+
+          <a href="#" class="button details__explore-link">Explore</a>
+        </secion>
+
+        <secion class="contacts main__contacts" id="contacts">
+          <h2 class="contacts__title">Contact us</h2>
+
+          <form
+            name="contact-form"
+            class="contact-us-form"
+            onsubmit="this.reset(); return false;"
+          >
+            <ul class="contact-us-form__list">
+              <li class="contact-us-form__item">
+                <input
+                  type="text"
+                  name="username"
+                  placeholder="Name"
+                  class="contact-us-form__input input"
+                  required
+                >
+              </li>
+
+              <li class="contact-us-form__item">
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="contact-us-form__input input"
+                  required
+                >
+              </li>
+
+              <li class="contact-us-form__item">
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us-form__textarea input"
+                  required
+                ></textarea>
+              </li>
+            </ul>
+
+            <button type="submit" class="contact-us-form__submit-button button">Send</button>
+          </form>
+
+          <ul class="contacts__list">
+            <li class="contacts__item">
+              <p class="contacts__info">
+                <span class="contacts__field">Phone</span>
+                <a href="tel:+1 234 5555-55-55" class="contacts__value contacts__value--hover">+1 234 5555-55-55</a>
+              </p>
+            </li>
+
+            <li class="contacts__item">
+              <p class="contacts__info">
+                <span class="contacts__field">Email</span>
+                <a href="mailto:hello@miami.com" class="contacts__value contacts__value--hover">hello@miami.com</a>
+              </p>
+            </li>
+
+            <li class="contacts__item">
+              <p class="contacts__info">
+                <span class="contacts__field">Address</span>
+                <a
+                  href="https://maps.app.goo.gl/KHdnyqRrUhy9NHYs7"
+                  class="contacts__value contacts__value--hover"
+                  target="_blank"
+                >
+                  400 first ave.
+                  <br>suite 700
+                  <br> Minneapolis, MN 55401
```

Comment:
Change text color on hover for phone, email, and address and you can remove your previous dynamic hover effect

[CHECKLIST](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md)